_posts/2015-12-16-API-Blueprint-Future.md

```diff
@@ -0,0 +1,100 @@
+---
+title: This is going to be Awesome
+excerpt: On the future of API Blueprint
+layout: post
+date: 2015-12-16 00:00:00 +0100
+author: zdenek
+published: true
+comments: true
+---
+
+What would be API Blueprint like if it started today? Different vocabulary, features or grammar concepts?
+
+The truth is, in past three years, API Blueprint was putting its flag in the sand. Carving out its concepts and defining the emerging API design space.
+
+From its beginnings API Blueprint stood out for its simplicity and collaborative nature. The aim has always been to create a simple, concise yet powerful language. A language in which everybody can think about the design of an API.
+
+I am still shocked how much these core values resonated with the community. It was the bet we took. To ask programmers and architects to write in plain text instead of JSON or XML. To not generate the API description from code but instead to take the effort and scope out an API definition before coding.
+
+After the three years I can say our bet was well worth taking. API Blueprint is now growing faster then ever. It has way over 3000 star gazers on [GitHub][1] and almost 200 000 blueprints just in Apiary itself. It's now used everywhere from IoT devices to banks.
+
+### The most important feature
+
+So where is API Blueprint going next? Before looking into the future let's start with the presence.
+
+In 2015, we have spent most of the year building only one feature — the description of body parameters. Also know as the [MSON syntax][2].
```

Comment:
"feature—the description of body parameters, also known as..."

Remove spaces around em dash and include final clause (currently a sentence fragment)